### ISSUE TITLE
Error on login / ReloadUI after game restart with timed awards active

### ISSUE THREAD

**Issue by ElmRR** [Mordonus/RaidOps#311]
Hi,

I'm seeing this error after relogging with timed awards still active. Upon logging back in RaidOps would throw this up as soon as the UI tried to load even when completely restarting the game:

...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\DKP.lua:792: attempt to index field 'wndActivity' (a nil value)
stack trace:
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\DKP.lua:792: in function 'UndoPopulate'
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\DKP.lua:677: in function 'UndoAddActivity'
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\DKP.lua:2003: in function 'TimeAwardAward'
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\DKP.lua:1904: in function 'TimeAwardStart'
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\DKP.lua:1932: in function 'TimeAwardRestore'
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\DKP.lua:485: in function <...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\DKP.lua:327>

**Comment by Drutol** [comment]
I believe that this is caused by 'Saved timed award undo' , You can disable this as a temporary solution before I make v2.27 stable and usable.

**Comment by ElmRR** [comment]
Thanks for the advice - will give this a go tonight.

Thanks for all the hard work, support and quick responses as well :)

**Comment by Drutol** [comment]
You can also add this line to DKP.lua after line 791.

> if not self.wndActivity then return end

It should look like this:

```
function DKP:UndoPopulate()
    if not self.wndActivity then return end
    local grid = self.wndActivity:FindChild("Grid")
    grid:DeleteAll()
    for k,activity in ipairs(tUndoActions) do
        grid:AddRow(k)
        grid:SetCellData(k,1,k)

```

**Comment by Hysteriana** [comment]
cant confirm, did several reloadui while time award timer running, no problems at all

**Comment by Drutol** [comment]
This is fairly rare issue , timer has been saved exactly when it was going to reset. Timed award module is initialized before undo module , I just need to adjust this init order.

**Comment by Drutol** [comment]
Fixed in v2.27
